### PR TITLE
fix test_docstrings significant figures

### PR DESCRIPTION
**Pull request by juliagsy**

### CODE REVIEW

File: ivy_tests/test_ivy/helpers.py [hunk 1]
```diff
@@ -413,20 +420,64 @@ def docstring_examples_run(*, fn, from_container=False, from_array=False):
...
+                " != ",
+                doc_v,
+                "\n",
+            )
+            ivy.warn(
+                "Output is unequal to the docstrings output: %s" % fn_name, stacklevel=0
+            )
+            break
+    return docstr_result
+    # if not (output == parsed_output):
```
Comment (djl11) [inline ivy_tests/test_ivy/helpers.py:467]: No need to keep the old version commented-out, just remove this.

File: ivy_tests/test_ivy/helpers.py [hunk 2]
```diff
@@ -413,20 +420,64 @@ def docstring_examples_run(*, fn, from_container=False, from_array=False):
...
-            "\noutput in docs :",
-            parsed_output,
-        )
-        ivy.warn(
-            "Output is unequal to the docstrings output: %s" % fn_name, stacklevel=0
-        )
-        return False
-    return True
+    sig_fig = float("1e-" + str(num_sig_fig))
+    # if "ivy.array(" in output:
```
Comment (djl11) [inline ivy_tests/test_ivy/helpers.py:424]: remove this?

**Review by djl11 — CHANGES_REQUESTED** [review]
Generally looks good, very small changes requested. Otherwise, provided it all works correctly, then I'm happy for this to be merged. Given that you have worked on this a lot, are you happy with these changes @RickSanchezStoic?

File: ivy_tests/test_ivy/helpers.py (hunk 1 above)
Comment (juliagsy) [reply ivy_tests/test_ivy/helpers.py]: ahh yes, was going through the last look before removing them hehe, just pushed the latest version without them!

**Comment by RickSanchezStoic** [comment]
Did you try running this on any existing failures @juliagsy  and verify once? It looks good to me!

**Comment by juliagsy** [comment]
@RickSanchezStoic I've run it before you skipped the failures, there isn't additional failures! It's either tests passing because values are being compared now, or tests failing as previous due to the docstring itself

**Comment by RickSanchezStoic** [comment]
It looks all good then! You may merge it.

**Pull request merged**